src/argilla/client/sdk/v1/datasets/models.py

```diff
@@ -112,7 +112,7 @@ class FeedbackQuestionModel(BaseModel):
 class FeedbackMetadataPropertyModel(BaseModel):
     id: UUID
     name: str
-    description: Optional[str] = None
+    title: Optional[str] = None
```

Comment:
If we are using this model to represent information from the API then I guess then `title` is not optional (in the same way that is not for `FeedbackFieldModel` and `FeedbackQuestionModel`

```suggestion
    title: str
```